codec/src/test/java/io/netty/handler/codec/compression/SnappyFrameDecoderTest.java

```diff
@@ -67,7 +70,7 @@ public void testReservedSkippableBeforeStreamIdentifier() throws Exception {
             -0x7f, 0x06, 0x00, 0x00, 's', 'n', 'e', 't', 't', 'y'
         });
 
-        channel.writeInbound(in);
+        assertFalse(channel.writeInbound(in));
```

Comment:
I think it makes more sense to omit this assert in the expected exception cases?

Reply:
You are right ... @Scottmitch can you adjust ?